android/src/com/frostwire/android/core/ConfigurationDefaults.java

```diff
@@ -102,6 +102,7 @@ private void load() {
 
         defaultValues.put(Constants.PREF_KEY_NETWORK_ENABLE_DHT, true);
         defaultValues.put(Constants.PREF_KEY_NETWORK_USE_MOBILE_DATA, true);
+        defaultValues.put(Constants.PREF_KEY_NETWORK_USE_WIFI_ONLY, false);
```

Comment:
lets not add another setting, this can already be handled with USE_MOBILE_DATA == false